setup.py

```diff
@@ -171,8 +171,15 @@ def __str__(self):
               ),
     Extension(
         'mprans.cRANS3PSed',
-        ['proteus/mprans/RANS3PSed.cpp', 'proteus/mprans/RANS3PSed2D.cpp'],
-        depends=['proteus/mprans/RANS3PSed.h','proteus/mprans/RANS3PSed2D.h', 'proteus/ModelFactory.h', 'proteus/CompKernel.h'],
+        ['proteus/mprans/cRANS3PSed.cpp'],
+        depends=['proteus/mprans/RANS3PSed.h','proteus/ModelFactory.h', 'proteus/CompKernel.h'],
+        include_dirs=get_xtensor_include(),
+        extra_compile_args=PROTEUS_OPT+['-std=c++14'],
+        language='c++'),
+    Extension(
+        'mprans.cRANS3PSed2D',
```

Comment:
Ah you replaced `RANS3PSed.cpp` with `cRANS3PSed.cpp` here (same for the 2D file) so the modules are not defined twice. Therefore, the `RANS3PSed.cpp` and `RANS3PSed2D.cpp` are not used anymore and could be deleted. However I would be in favor of doing the contrary to keep naming consistency with other modules.